reframe/core/logging.py

```diff
@@ -378,6 +380,24 @@ def _update_check_extras(self):
         if self.check.job:
             self.extra['check_jobid'] = self.check.job.jobid
 
+        try:
+            import pwd
+            self.extra['username'] = pwd.getpwuid(os.geteuid()).pw_name
+        except:
+            try:
+                import getpass
+                self.extra['username'] = getpass.getuser()
+            except:
```

Comment:
Bare except clauses are disallowed by PEP8. Either be more precise or catch `Exception` or `BaseException` (not this case).